src/Exception/InvalidArgumentException.php

```diff
@@ -36,7 +36,7 @@ class InvalidArgumentException extends DriverInvalidArgumentException implements
      * @param string|string[] $expectedType Expected type
      * @return self
      */
-    public static function invalidType($name, $value, $expectedType)
+    public static function invalidType(string $name, $value, $expectedType)
```

Comment:
Noted that we can't use union types for `$expectedType`, as that [requires PHP 8.0](https://php.watch/versions/8.0/union-types).